tests/unit/plugins/modules/network/nso/nso_module.py

```diff
@@ -15,7 +15,8 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-from __future__ import (absolute_import, division, print_function)
+from __future__ import absolute_import, division, print_function
```

Comment:
```suggestion
from __future__ import absolute_import, division, print_function
```
earlier you pasted this line with parentheses as well:)

Reply:
I usually copied them from similar files :)